misk/web/tabs/web-actions/src/rewrite/WebActionParameters.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { Card, H5, HTMLTable } from "@blueprintjs/core"
+import { ParameterMetaData } from "./types"
+import WebActionParameter from "./WebActionParameter"
+
+interface Props {
+  parameters: ParameterMetaData[]
+}
+
+export default function WebActionParameters({ parameters }: Props) {
+  if (parameters.length == 0) {
+    return null
+  }
+
+  return (
+    <Card style={{ marginTop: "12px", padding: "12px" }}>
+      <H5> Request Parameters </H5>
+      <HTMLTable>
```

Comment:
Is there any way to get the space below here to be less big? It looks kinda bleh to have
![image](https://user-images.githubusercontent.com/4494830/115408879-3aaa5900-a1bf-11eb-8de8-32a800df6a72.png)

Reply:
Addressed in: https://github.com/cashapp/misk/pull/1850/commits/c710c167ddcebbf73c5b752e103294c98ed6c3c5